docs/sites.yml

```
@@ -9611,3 +9611,23 @@
   built_by: KiwiSprout
   built_by_url: https://kiwisprout.nz/
   featured: false
+- title: DG Recruit
+  main_url: https://dgrecruit.com
```

You'll want to add `url` to both of these as well!

```suggestion
  url: https://dgrecruit.com
  main_url: https://dgrecruit.com
```